translations/zh-cn/README.md

```diff
@@ -0,0 +1,257 @@
+<p align="center">
+  <img src="https://i.imgur.com/NVRZLHv.png" width="640" alt="quicklink">
+  <br>
+  <a href="https://www.npmjs.org/package/quicklink"><img src="https://img.shields.io/npm/v/quicklink.svg?style=flat" alt="npm"></a>
+  <a href="https://unpkg.com/quicklink"><img src="https://img.badgesize.io/https://unpkg.com/quicklink/dist/quicklink.js?compression=gzip" alt="gzip size"></a>
+  <a href="https://www.npmjs.com/package/quicklink"><img src="https://img.shields.io/npm/dt/quicklink.svg" alt="downloads" ></a>
+  <a href="https://travis-ci.org/GoogleChromeLabs/quicklink"><img src="https://travis-ci.org/GoogleChromeLabs/quicklink.svg?branch=master" alt="travis"></a>
+</p>
+
+# quicklink
+
+> 可以在空闲时间预获取页面可视区域（以下简称视区）内的链接，加快后续加载速度。
+
+## 工作原理
+
+Quicklink 通过以下方式加快后续页面的加载速度：
+
+- **检测视区中的链接**（使用 [Intersection Observer](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API)）。
+- **等待浏览器空闲**（使用 [requestIdleCallback](https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback)）。
```

Comment:
Is it worth updating docs links to the translations on MDN too? 😄 e.g. https://developer.mozilla.org/zh-CN/docs/Web/API/Window/requestIdleCallback